sqlglot/expressions.py

```diff
@@ -2314,6 +2314,16 @@ class Fetch(Expression):
     }
 
 
+class Grant(Expression):
+    arg_types = {
+        "privilages": True,
+        "securable_kind": False,
```

Comment:
Should this just be `kind`? Seems inconsistent with `Create`: we don't have `creatable_kind` there, it's implied.

Reply:
Right, this was initially named so to distinguish from `principal_kind` but later on I split that in the separate node `GrantPrincipal`.

It's not consistent with the codebase but it's also not directly connected to `GRANT` (in contrast to `CREATE`) since there's many different constructs at play here (securables, principals etc). I'll revert this to `kind` though for simplicity.